makefile

```diff
@@ -22,13 +25,25 @@ PREFIX=/usr/local
 EXEC_CLI=fsqlf
 EXEC_GUI=wx_fsqlf
 CC=gcc
-CFLAGS+=-m32
+ARCH=-m32
 CXX=g++
 CXXFLAGS+= `wx-config --cxxflags`
 LDFLAGS+= `wx-config --libs`
+DEFINES=linux
```

Comment:
I added that line because Mac OS X doesn't support `fmemopen` and `open_memstream` which were two handy functions that let me create the `translate_string` function.

In looking at that line, it should probably be:
     
    DEFINES=-Dlinux"